HPE3PARPSToolkit-CLI.psm1

```diff
@@ -34380,11 +34399,8961 @@ Function Stop-3parWsapi()
+##///////////////////////////////////////////////////////////////
+##///////////////////////////////////////////////////////////////
+##v2.3 CIMF Sprint 11
+##///////////////////////////////////////////////////////////////
+##///////////////////////////////////////////////////////////////
+
+
+##########################################################################
+######################### FUNCTION Set-3parDomain #########################
+##########################################################################
+Function Set-3parDomain()
+{
+<#
+  .SYNOPSIS
+   Set-3parDomain Change currentdomain CLI environment parameter.
```

Comment:
Showing currentdomain as a single word. Change it to "current domain"
Corrected sentence: Set-3parDomain changes the current domain CLI environment parameter.

Reply:
Updated.